packages/patternfly-4/react-core/src/components/ClipboardCopy/ClipboardCopy.tsx

```diff
@@ -91,6 +91,12 @@ export class ClipboardCopy extends React.Component<ClipboardCopyProps, Clipboard
     toggleAriaLabel: 'Show content'
   };
 
+  componentDidUpdate = (prevProps: ClipboardCopyProps, prevState: ClipboardCopyState) => {
+    if (prevProps.children !== this.props.children) {
```

Comment:
This check will work because we technically only support `children` being a string (despite it's type being ReactNode).